.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature request
-about: ''
+about: 'Suggest an idea for improving Compose Multiplatform'
 title: ''
 labels: ''
```

Comment:
```suggestion
labels: 'enhancement'
```

Reply:
Yes, we have it.

Also, changed name and about to the description of this tag

Reply:
Also added `bug` to bug reports